### PR TITLE
Renamed almighty to fabric8-services

### PR DESCRIPTION
**Pull request by hectorj2f**
@maxandersen : could you have a look at these changes ? I believe you know which emails of url should use.

### CODE REVIEW

File: _blueprints/build_concept.adoc [hunk 1]
```diff
@@ -86,11 +86,11 @@ image::projectbuildconfigdataflow.png["Project Build Configuration Data Flow",fl
...
+*For private GitHub repositories*.footnote:[ see mailing list discussion here: https://www.redhat.com/archives/fabric8-services-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw[https://www.redhat.com/archives/fabric8-services-public/2016-October/msg00110.html&sa=D&ust=1477567580676000&usg=AFQjCNEA1AUUb8L54soCYBUBN5V75dq3Hw]], the project configuration contains a ssh secret key that is used to access the private repository. The user gets a notification with the public key and is noted that he has to add the public key to the deploy keys on his git provider. As this procedure is not specific for OpenShift or GitHub, the needed UI dialogs are also generic. As an option, if the repository and/or build provider supports this, the key exchange can be done automatically via a provided API.footnote:[ For example: https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key[https://developer.github.com/v3/repos/keys/#add-a-new-deploy-key]].
```
Comment (maxandersen) [inline _blueprints/build_concept.adoc]: that url seems broken and I don't recall what it was about ;/ better to remove it than try rename it to another 404 link

File: _blueprints/build_concept.adoc [hunk 2]
```diff
@@ -130,31 +130,31 @@ As the provider-specific configuration is contained in the editable configuratio
 
 The ALM Build Service contains a Service Provider Interface (SPI) that interconnects the ALM Build Service to the (possibly) 3rd party build providers using a well defined interface. The interface consists of a technical REST api, a procedure for registering new build providers to the ALM Build Service and a set of interactions modelled as REST flows for the various user operations (see above).
 
-==== SPI Architecture.footnote:[ Discussion on different approaches here: https://www.google.com/url?q=https://www.redhat.com/archives/almighty-public/2016-October/msg00128.html&sa=D&ust=1478189307991000&usg=AFQjCNHaozEeVwWN0yJqaso6RREngoUtyA[https://www.google.com/url?q=https://www.redhat.com/archives/almighty-public/2016-October/msg00128.html&sa=D&ust=1478189307991000&usg=AFQjCNHaozEeVwWN0yJqaso6RREngoUtyA]]
```
Comment (maxandersen) [inline _blueprints/build_concept.adoc]: same thing  - urls are broken;  not sure why ;/

File: _blueprints/build_concept.adoc [hunk 3]
```diff
@@ -130,31 +130,31 @@ As the provider-specific configuration is contained in the editable configuratio
...
 
 The SPI is modelled following a “consumer” model. The SPI defines a set of REST operations that are consumed from the actual build provider. So the SPI expects the build provider (or an intermediate party) to provide this interface for consumption.
 
-==== Logical SPI Operations.footnote:[ Preliminary specification here: https://www.google.com/url?q=https://github.com/tnozicka/almighty-devdoc/blob/add-build-spi-definition/design/build_spi.adoc&sa=D&ust=1478189307979000&usg=AFQjCNGzUBbihwXQH4rvWNoRfTY0w48kBA[https://www.google.com/url?q=https://github.com/tnozicka/almighty-devdoc/blob/add-build-spi-definition/design/build_spi.adoc&sa=D&ust=1478189307979000&usg=AFQjCNGzUBbihwXQH4rvWNoRfTY0w48kBA]]
+==== Logical SPI Operations.footnote:[ Preliminary specification here: https://www.google.com/url?q=https://github.com/tnozicka/fabric8-devdoc/blob/add-build-spi-definition/design/build_spi.adoc&sa=D&ust=1478189307979000&usg=AFQjCNGzUBbihwXQH4rvWNoRfTY0w48kBA[https://www.google.com/url?q=https://github.com/tnozicka/fabric8-devdoc/blob/add-build-spi-definition/design/build_spi.adoc&sa=D&ust=1478189307979000&usg=AFQjCNGzUBbihwXQH4rvWNoRfTY0w48kBA]]
```
Comment (maxandersen) [inline _blueprints/build_concept.adoc]: this looks like too agressive an search/replace. just keep it as is.

File: cico_build_deploy.sh [hunk 4]
```diff
@@ -39,16 +39,16 @@ echo 'CICO: Deploying to github pages'
...
 git checkout -b gh-pages origin/gh-pages
 
-# Move over generated content to new repo 
+# Move over generated content to new repo
 cp -rfv _site/* .
 
 # Update new repo with generated code
 git config user.name "ci.centos.org"
-git config user.email "almighty-public@redhat.com"
+git config user.email "fabric8-services-public@redhat.com"
```
Comment (maxandersen) [inline cico_build_deploy.sh]: could use fabric8@googlegroups.com

File: ux/walkthroughs/posting-completed-designs.adoc [hunk 5]
```diff
@@ -38,11 +38,11 @@ _*(artifacts include materials related to the design, discovery, workflows, wire
...
-* On the Almighty channel using @here, the *InVision link*, and a brief message.
+* On the Fabric8 services UI/UX channel using @all, the *InVision link*, and a brief message.
+* On the Fabric8 services channel using @here, the *InVision link*, and a brief message.
 . Post to email lists
-* Internal: Post here any sensitive materials to share with the internal group and/or when a design is in progress and you are working with development on specific solution issues:  mailto:devtools-almighty@redhat.com[devtools-almighty@redhat.com]
-* External: Post here when you have vetted the design through UXD and are ready for broader/public/community feedback: mailto:almighty-public@redhat.com[almighty-public@redhat.com]
+* Internal: Post here any sensitive materials to share with the internal group and/or when a design is in progress and you are working with development on specific solution issues:  mailto:devtools-saas@redhat.com[devtools-saas@redhat.com]
+* External: Post here when you have vetted the design through UXD and are ready for broader/public/community feedback: mailto:devtools-saas@redhat.com[devtools-saas@redhat.com]
```
Comment (maxandersen) [inline ux/walkthroughs/posting-completed-designs.adoc]: devtools-saas@ are non public. use fabric8@googlegroups.com

**Comment by hectorj2f** [comment]
thanks for the review @maxandersen

File: _blueprints/build_concept.adoc (hunk 1 above)
Comment (hectorj2f) [reply _blueprints/build_concept.adoc]: Done

File: _blueprints/build_concept.adoc (hunk 2 above)
Comment (hectorj2f) [reply _blueprints/build_concept.adoc]: done

File: cico_build_deploy.sh (hunk 4 above)
Comment (hectorj2f) [reply cico_build_deploy.sh]: Added the email address

Comment (maxandersen) [reply cico_build_deploy.sh]: where did "fabric8-services-public@redhat.com" come from ?

there is fabric8@googlegroups.com and fabric8-planner@googlegroups.com...why now a fabric8 redhat.com one ?

Comment (hectorj2f) [reply cico_build_deploy.sh]: I use `fabric8@googlegroups.com` now https://github.com/fabric8-services/fabric8-devdoc/pull/129/files#diff-5bf05017b1ea493887ae831ae52f78ebL47

The email came from a hardcoded sed command, that is why I asked for a revision for the emails, and other details.

Comment (maxandersen) [reply cico_build_deploy.sh]: gotcha - I for some reason read it as you updated the mail and then in the PR it listed the old one. 

Gotcha!

**Comment by alexeykazakov** [comment]
Closing as outdated.

**Pull request closed without merging**